vignettes/population_usage.Rmd

```diff
@@ -0,0 +1,256 @@
+---
+title: "Estimating population usage from input distribution"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Estimating population usage from input distribution}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(netz)
+library(ggplot2)
+set.seed(1234)
+```
+
+In [malariasimulation](https://mrc-ide.github.io/malariasimulation/) we can implement the [bed net intervention](https://mrc-ide.github.io/malariasimulation/articles/VectorControl.html) by specifying
+the % of the population who are distributed a bed net (and assumed to use it) at
+given time points.
+
+Often it is helpful to understand the resulting population levels of bed net usage from a set of 
+bed net distributions over time. The % of individuals protected by a net at any time point is a 
+function of the cumulative distributions that have occurred, the durations of net retention and the
+correlation of recipients between rounds.
+
+In netz, we can simulate population usage and examine how varying distribution size and frequency,
+net retention and recipient correlation impacts our estimates. Please note that simulations of usage
+are stochastic.
+
+Let's start with a simple distribution model:
+
+* Distribution cycle: A mass distribution once every 3 years
+* Distribution size: 70% of the population covered
+* Net retention: Assume net retention decays exponentially with a half life of 5 years
+* Correlation: Recipients are correlated each round
+
+```{r, fig.width = 7, fig.height = 4}
```

Comment:
Might be nice to add an estimate of the length/memory it takes to run these simulations - not essential, but nice :)

Reply:
I think we're ok here as they're pretty fast and not too memory intensive.